server/src/main/java/org/elasticsearch/index/fielddata/plain/DocValuesIndexFieldData.java

```diff
@@ -87,7 +88,7 @@ public Builder scriptFunction(Function<SortedSetDocValues, ScriptDocValues<?>> s
                                        CircuitBreakerService breakerService, MapperService mapperService) {
             // Ignore Circuit Breaker
             final String fieldName = fieldType.name();
-            if (BINARY_INDEX_FIELD_NAMES.contains(fieldName)) {
+            if (BINARY_INDEX_FIELD_NAMES.contains(fieldName)|| fieldType.getClass() == RangeFieldMapper.RangeFieldType.class) {
```

Comment:
Note to self: @polyfractal and I kicked this around, and we think the correct short term solution (i.e. for this PR) is to add a `rangeType()` method or similar to flag this, similar to how numeric and script work, rather than relying on the field type.